client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java

```diff
@@ -154,6 +154,9 @@ static Request bulk(BulkRequest bulkRequest) throws IOException {
         parameters.withRefreshPolicy(bulkRequest.getRefreshPolicy());
         parameters.withPipeline(bulkRequest.pipeline());
         parameters.withRouting(bulkRequest.routing());
+        if (bulkRequest.requireAlias() != null) {
```

Comment:
I think this `if` is not required.

How?

https://github.com/opensearch-project/OpenSearch/blob/04a417a7328ef7d4c501508703898ff2b477f302/client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java#L1019

Reply:
That if condition is required because the `bulkRequest.requireAlias()` method is returning Boolean wrapper class instead of boolean keyword. If we do not explicitly pass requireAlias with the request, it will pass on null value to the `withRequireAlias` method since it is expecting primitive type boolean parameter.

https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/action/bulk/BulkRequest.java#L427

https://github.com/opensearch-project/OpenSearch/blob/04a417a7328ef7d4c501508703898ff2b477f302/client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java#L1018